OpenQA/Isotovideo/Utils.pm

```diff
@@ -158,6 +157,8 @@ sub handle_generated_assets ($command_handler, $clean_shutdown) {
         bmwqemu::diag "Requested to force the publication of '$name'";
         push @toextract, _store_asset($i, $name, 'assets_public');
     }
+    # Nothing to do here if we don't have any assets to extract
+    return unless @toextract;
```

Comment:
I think we aren't using `return` without a value.
You should return `$return_code`, however, that is already done after the loop (which isn't executed if the array is empty anyway), so I don't see how that would change anything.